src/OpenTelemetry.Exporter.Geneva/README.md

```diff
@@ -157,6 +157,57 @@ values.
   * `ILogger<MyPartner.Product.Thing>`: This is marked as pass-through ("*") so
     it will be sanitized as "MyPartnerProductThing" table name
 
+#### How to configure GenevaExporterOptions using dependency injection
+
+##### Tracing
+
+> [!NOTE]
+> In this example named options ('GenevaTracing') are used. This is because
+> `GenevaExporterOptions` is shared by both logging & tracing. In a future
+> version named options will also be supported in logging so it is recommended
+> to use named options now for tracing in order to future-proof this code.
+
+```csharp
+// Step 1: Turn on tracing and register GenevaTraceExporter.
+services.AddOpenTelemetry()
+    .WithTracing(builder => builder
+        .AddGenevaTraceExporter(
+            "GenevaTracing", // Tell GenevaTraceExporter to retrieve options using the 'GenevaTracing' name
+            _ => { }));
+
+// Step 2: Use Options API to configure GenevaExporterOptions using services
+// retrieved from the dependency injection container
+services
+    .AddOptions<GenevaExporterOptions>("GenevaTracing") // Register options with the 'GenevaTracing' name
+    .Configure<IConfiguration>((exporterOptions, configuration) =>
+    {
+        exporterOptions.ConnectionString = configuration.GetValue<string>("OpenTelemetry:Tracing:GenevaConnectionString")
+            ?? throw new InvalidOperationException("GenevaConnectionString was not found in application configuration");
+    });
+```
+
+##### Logging
+
+```csharp
+// Step 1: Turn on logging.
+builder.Logging.AddOpenTelemetry();
+
+// Step 2: Use Options API to configure OpenTelemetryLoggerOptions using
+// services retrieved from the dependency injection container
+services
+    .AddOptions<OpenTelemetryLoggerOptions>()
+    .Configure<IConfiguration>((loggingOptions, configuration) =>
```

Comment:
"loggingOptions" or "loggerOptions" or "logging"?

Reply:
I just switched it to "loggerOptions" which seems more consistent with "exporterOptions"